modules/cron4s/build.sbt

```
@@ -1,9 +1,8 @@
 name := "pureconfig-cron4s"
 
-crossScalaVersions ~= { _.filterNot(_.startsWith("2.13")) }
+crossScalaVersions := List("2.13.1", "2.12.9")
```

Just to make it easier to define versions in a single location, can you change this to filter scala versions starting with 2.11?

```suggestion
crossScalaVersions ~= { _.filterNot(_.startsWith("2.11")) }
```